apps/testingaccessibility/.env.development

```diff
@@ -1,3 +1,19 @@
+NEXTAUTH_URL=http://localhost:3013
+NEXTAUTH_SECRET=f8e1297311697b20fdac1fb9dfc55a2a # https://generate-secret.now.sh/32
+
+# next/client for local hasura
+HASURA_GRAPHQL_ENDPOINT=http://localhost:8080/v1/graphql
+HASURA_ADMIN_SECRET=admin_secret
+
+# hasura local env -> `docker-compose up`
+PG_DATABASE_URL=postgres://postgres:postgrespassword@postgres:5432/postgres
+HASURA_GRAPHQL_METADATA_DATABASE_URL=${PG_DATABASE_URL}
+HASURA_GRAPHQL_ENABLE_CONSOLE="false"
+HASURA_GRAPHQL_DEV_MODE="true"
+HASURA_GRAPHQL_ENABLED_LOG_TYPES=startup, http-log, webhook-log, websocket-log, query-log
+HASURA_GRAPHQL_ADMIN_SECRET=${HASURA_ADMIN_SECRET}
+HASURA_GRAPHQL_JWT_SECRET='{"key": "${NEXTAUTH_SECRET}","type": "HS256"}'
```

Comment:
moved all of this out of .env.local since it is generally useful for local dev

also learned about the `${VARIABLE}` syntax which is handy